store/src/main/java/com/nytimes/android/external/store/base/impl/ParsingStoreBuilder.java

```diff
@@ -81,13 +84,22 @@ public Raw call() throws Exception {
 
     @NonNull
     public ParsingStoreBuilder<Raw, Parsed> parser(final @NonNull Func1<Raw, Parsed> parser) {
-        this.parser = parser;
+        this.parsers.clear();
+        this.parsers.add((Parser<Raw, Parsed>) parser);
         return this;
     }
 
     @NonNull
     public ParsingStoreBuilder<Raw, Parsed> parser(final @NonNull Parser<Raw, Parsed> parser) {
```

Comment:
Any reason to clear when adding another parser? Not saying it should exist just trying to figure out logic behind not allowing someone to call builder.parser().parser().build() and expecting 2 parsers to be added?